code/modules/vore/eating/digest_act_vr.dm

```diff
@@ -76,8 +77,6 @@
 			var/obj/item/device/pda/P = src
 			if(P.id)
 				P.id = null
-		for(var/mob/living/voice/V in possessed_voice) // Delete voices.
```

Comment:
Better to comment these lines out with //CHOMPedit comments instead of deleting, otherwise it may cause confusion in a merge conflict